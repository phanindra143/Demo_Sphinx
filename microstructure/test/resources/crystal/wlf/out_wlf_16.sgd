*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   104.1110
  !k_ij(1,2)=     0.6205
  !k_ij(2,2)=   105.4436
  !k_ij(1,3)=     0.7441
  !k_ij(2,3)=    -0.3373
  !k_ij(3,3)=   105.6138
MP,Kxx,1,  104.11102
MP,Kyy,1,  105.44365
MP,Kzz,1,  105.61379
MP,Kxx,2,  105.44365
MP,Kyy,2,  104.11102
MP,Kzz,2,  105.61379
MP,Kxx,3,  105.44365
MP,Kyy,3,  104.11102
MP,Kzz,3,  105.61379
MP,Kxx,4,  104.11102
MP,Kyy,4,  105.44365
MP,Kzz,4,  105.61379
MP,Kxx,5,  104.11102
MP,Kyy,5,  105.44365
MP,Kzz,5,  105.61379
MP,Kxx,6,  105.44365
MP,Kyy,6,  104.11102
MP,Kzz,6,  105.61379
MP,Kxx,7,  105.44365
MP,Kyy,7,  104.11102
MP,Kzz,7,  105.61379
MP,Kxx,8,  104.11102
MP,Kyy,8,  105.44365
MP,Kzz,8,  105.61379
MP,Kxx,9,  105.61379
MP,Kyy,9,  104.11102
MP,Kzz,9,  105.44365
MP,Kxx,10,  104.11102
MP,Kyy,10,  105.61379
MP,Kzz,10,  105.44365
MP,Kxx,11,  104.11102
MP,Kyy,11,  105.61379
MP,Kzz,11,  105.44365
MP,Kxx,12,  105.61379
MP,Kyy,12,  104.11102
MP,Kzz,12,  105.44365
MP,Kxx,13,  105.61379
MP,Kyy,13,  104.11102
MP,Kzz,13,  105.44365
MP,Kxx,14,  104.11102
MP,Kyy,14,  105.61379
MP,Kzz,14,  105.44365
MP,Kxx,15,  104.11102
MP,Kyy,15,  105.61379
MP,Kzz,15,  105.44365
MP,Kxx,16,  105.61379
MP,Kyy,16,  104.11102
MP,Kzz,16,  105.44365
MP,Kxx,17,  105.44365
MP,Kyy,17,  105.61379
MP,Kzz,17,  104.11102
MP,Kxx,18,  105.61379
MP,Kyy,18,  105.44365
MP,Kzz,18,  104.11102
MP,Kxx,19,  105.61379
MP,Kyy,19,  105.44365
MP,Kzz,19,  104.11102
MP,Kxx,20,  105.44365
MP,Kyy,20,  105.61379
MP,Kzz,20,  104.11102
MP,Kxx,21,  105.44365
MP,Kyy,21,  105.61379
MP,Kzz,21,  104.11102
MP,Kxx,22,  105.61379
MP,Kyy,22,  105.44365
MP,Kzz,22,  104.11102
MP,Kxx,23,  105.61379
MP,Kyy,23,  105.44365
MP,Kzz,23,  104.11102
MP,Kxx,24,  105.44365
MP,Kyy,24,  105.61379
MP,Kzz,24,  104.11102
*endif
