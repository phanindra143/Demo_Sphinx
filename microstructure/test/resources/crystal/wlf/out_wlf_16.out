            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\8
             3.00000      -1626622.02718          6005.36176         -7725.94309
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000          -145.35068           -36.70951
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000          8496.90891      -1647441.79547         -4921.28021
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000          -178.65039         15625.00000            61.41405
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -1903.05871           617.46409      -1650102.26035
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -94.98730            81.08909         15625.00000
  /%
